lib/shanty/plugin.rb

```diff
@@ -41,6 +41,10 @@ def projects
       end
     end
 
+    def artifacts(*)
```

Comment:
I suspect this should actually be `def artifacts(_)` as it should only ever take one project to return the artifacts for at a time.